prow/plugins/plugins.go

```diff
@@ -201,6 +202,17 @@ type Blockade struct {
 	Explanation string `json:"explanation,omitempty"`
 }
 
+type Approve struct {
+	// Repos is either of the form org/repos or just org.
+	Repos []string `json:"repos,omitempty"`
```

Comment:
Do we need this?

Reply:
I think so. The approve plugin will be enabled on multiple repos and they won't all want the same settings. For example, most repos won't want to require an associated issue, but k/k will.

Reply:
It would be nice to fix this problem (different config between repos for a plugin) in a generic way but that can be a follow-up issue.